### ISSUE TITLE
[agent instructions][claude][task] Human-checkpoint batch step 1 - A11 audition + A18 item 2

### ISSUE THREAD

**Issue by dudarenok-maker** [dudarenok-maker/Castwright#2994]
Step 1 of the agent-prefix + human-checkpoint batch (#2978), on-box
acceptance register campaign (#2435). First step — starts the chain.

## What

Two rows: **A11** (`docs/testing/onbox-acceptance-register.md:1590`) — A/B
"current vs proposed" voice audition — and **A18, item 2 only**
(`:1847`, the "Latent equivalence" bullet). A18's items 1, 3 are already
DISCHARGED and item 4 is covered by a different, already-filed chain
(#2950 — do not touch it here).

### A11

"GPU audition validation owed" since 2026-06-01 (the oldest debt in the
whole register). Plan 161's A/B redesign: **Cancel must leave the live
`.pt` untouched**, plus an audible delta on approve. Read the full row —
it's short, there's no wave history yet.

### A18 item 2

Read the full row (`:1847-1896`) — it's long, with real history. Item 2's
own text: latent **decode** equivalence between the patched and unpatched
`load_audio` was already measured bit-identical (max diff **0.0**) during PR
#1978's review, on a box that was hot-patched at the time. What remains is
the **audible** end — derive the same cloned voice with and without the
`patched_xtts_load_audio()` wrap on a box with genuinely **shared** FFmpeg,
and confirm equivalence. **The row's own "Needs" line says item 2 wants "a
box with a genuinely shared FFmpeg"** — and this dev box was reverted to
static-FFmpeg on 2026-07-31 (see item 1's discharge note, same row) and is
**not** that box anymore.

### A11 — Cancel-leaves-.pt-untouched (agent, mechanical)

1. Find the A/B audition flow (plan 161 — grep for "audition" in
   `src/components/`/`src/modals/` and the corresponding server route).
2. Trigger a proposed-voice audition against a real cast voice, note the
   `.pt` file's mtime/hash before, click **Cancel** (not Approve), and
   confirm the `.pt` file is byte-identical / unchanged (hash comparison,
   not just "still exists").

### A11 — audible delta on approve (stage for human)

3. Trigger the same audition flow, this time **Approve** it, and render a
   short line with both the **current** and **proposed** voice.
4. Stage both clips as separate files
   (`docs/testing/onbox-human-checkpoint-results/a11-current.wav`,
   `a11-proposed.wav`).
5. Also compute an `/embed` cosine similarity between the two clips as a
   supporting number (not a substitute for the ear check this row's own
   framing asks for — audition A/B delta is exactly the kind of thing the
   design intends a human to judge, so stage it, don't force a discharge).

### A18 item 2 — check the precondition before attempting anything

6. Confirm directly on this box: `import torchcodec` — does it fail (static-
   FFmpeg, as item 1's discharge note describes) or succeed? Don't trust the
   row text's date-stamped claim; check it live.
7. **If it fails** (expected, per the 2026-07-31 revert): this row's own
   precondition — a genuinely shared-FFmpeg box — is absent here. Do **not**
   attempt to reconfigure this shared box's FFmpeg install to manufacture
   one; that would risk every other lane and every other row that depends
   on the current static-FFmpeg state (item 1's own discharge explicitly
   relies on it). Record this as BLOCKED-ACQUISITION: "needs a genuinely
   shared-FFmpeg box, this one was deliberately reverted to static-FFmpeg on
   2026-07-31 and reconfiguring it would risk other rows/lanes" — no faked
   work, no staged evidence pretending otherwise.
8. **If it unexpectedly succeeds** (the box's FFmpeg state changed since the
   revert note was written): derive the same cloned voice with and without
   the patch wrap and diff the resulting audio files. Since decode
   equivalence is already proven bit-identical at the source, an
   audio-file-level diff (`ffprobe`/byte comparison) is sufficient here — no
   ear needed, because bit-identical decode input plus the same rendering
   pipeline implies bit-identical output; if the outputs differ, that's a
   genuine finding to report, not something to explain away.

## Acceptance

- A11: Cancel-untouched confirmed via hash comparison. Current/proposed
  clips staged as separate files with an embedding cosine number recorded,
  and the evidence file states plainly that the audible-delta judgment
  itself is the remaining human checkpoint.
- A18 item 2: the precondition check is run live (not assumed from the row
  text's date), and the outcome — blocked-acquisition or a real
  agent-executable equivalence check — is recorded with the actual command
  output.

## Key files

- `docs/testing/onbox-acceptance-register.md:1590-1594` (A11)
- `docs/testing/onbox-acceptance-register.md:1847-1896` (A18, all 4 items —
  item 2 is yours, items 1/3 are historical context, item 4 belongs to
  #2950)
- `docs/superpowers/specs/2026-07-31-xtts-clone-torchcodec-ffmpeg-design.md`
  §12 (A18's criteria doc)

## Not in scope

A18 items 1, 3, 4 (item 4 specifically belongs to the already-filed #2950 —
do not duplicate it). Any register edit — step 8 (#2985) does that.

## Where the work happens

Worktree `C:\Claude\Projects\wt-human-checkpoint-batch`, branch
`docs/docs-human-checkpoint-batch` (already created, dependencies installed,
husky active, `server/tts-sidecar/.venv`+`voices/` already junctioned from
the primary checkout). Ports: VITE 5363, API 8270, TTS 9190. Do not create
another worktree. Do not work in the primary checkout. Commit your evidence
file (`docs/testing/onbox-human-checkpoint-results/step1-a11-a18.md`, plus
the two staged `.wav` files) on this branch. Do not open a PR.

## Standing rules for this whole chain

- Never stop, kill, or restart any other process on this box — 59
  node/python/ollama processes from other lanes are live right now.
- Never touch `server/.env` outside this worktree, and never modify the
  shared box's real FFmpeg/torchcodec install to manufacture A18 item 2's
  precondition.
- Never `--no-verify`.
- This step touches no `server/**` source, only `docs/testing/**` evidence —
  but if a `server/**` file is staged for any reason, the commit gate is
  real regardless: a commit there can take 5-7 minutes, a push the same
  again, and this runtime kills a single foreground command at 30 seconds —
  launch `git commit`/`git push` detached and poll for an `EXIT=` sentinel,
  never foreground, never `--no-verify`.

## On completion — enable your successor YOURSELF, in this order

**1. Flip the successor to `Agent Todo` FIRST**, then **2.** post an
`AGENT DONE` comment, then **3.** close your issue.

```bash
gh api graphql -F owner=dudarenok-maker -F repo=Castwright -F num=2993 -f query='
  query($owner:String!,$repo:String!,$num:Int!){ repository(owner:$owner,name:$repo){
    issue(number:$num){ projectItems(first:10){ nodes{ id } } } } }' \
  --jq '.data.repository.issue.projectItems.nodes[0].id'

gh project item-edit --id <ITEM_ID> --project-id PVT_kwHOEOX6_c4Bcf9a \
  --field-id PVTSSF_lAHOEOX6_c4Bcf9azhXISes --single-select-option-id 9d9fa565
```

Refs #2978, #2435

**Comment by dudarenok-maker** [comment]
AGENT CLAIMED
Agent: claude

Claimed via scheduler fast path (todo). Starting work: A11 audition Cancel-untouched + audible-delta staging, and A18 item 2 precondition check, in worktree C:\Claude\Projects\wt-human-checkpoint-batch (branch docs/docs-human-checkpoint-batch).

**Comment by dudarenok-maker** [comment]
AGENT DONE
Agent: claude

Ran on-box in worktree wt-human-checkpoint-batch (branch docs/docs-human-checkpoint-batch), real local Qwen (qwen3-tts-0.6b) + real sidecar /embed, no mocks.

## A11 — A/B "current vs proposed" voice audition (plan 161)
Drove the redesign REST surface directly (same calls RedesignLibraryVoiceModal makes) against a real cast voice created for this check (voiceUuid d0nnbNL9Qe2QULJ4JeLSZ):
- Cancel-untouched confirmed via SHA-256 hash of the live `.pt`: identical (2870c22a...) before redesign, after POST /redesign staged a preview, and after POST /redesign/discard (Cancel). Staged preview files (qwen-<uuid>-preview.pt/.json/__master.wav) confirmed removed after discard.
- Asymmetry check: a second redesign + POST /redesign/promote (Approve) on the same voice did change the live `.pt` hash (to 6870566e...), confirming the Cancel-path hash stability is real and not an inert key.
- Audible delta staged for human judgment: current + proposed clips saved as docs/testing/onbox-human-checkpoint-results/a11-current.mp3 and a11-proposed.mp3. Supporting number (not a substitute for the ear check): cosine similarity between the two clips' real sidecar /embed (ECAPA-TDNN, 192-d) embeddings = 0.246, computed via ffmpeg-decoded 16kHz mono PCM posted to the live sidecar.

## A18 item 2 — cloned-voice derive latent equivalence, torchcodec precondition
Checked live rather than assumed from the row text's date: `import torchcodec` fails on this box with the same libtorchcodec_core4/5.dll load failure item 1's discharge note describes — this dev box is still genuinely static-FFmpeg (reverted 2026-07-31).

**BLOCKED-ACQUISITION**: item 2's own stated precondition — "a box with a genuinely shared FFmpeg" — is absent here. Per the ticket's own instruction, did not reconfigure this shared box's FFmpeg/torchcodec install to manufacture the precondition (would risk other rows/lanes, including A18 item 1's own discharge). No faked work.

Evidence file: docs/testing/onbox-human-checkpoint-results/step1-a11-a18.md (commit 832b7d5, pushed to docs/docs-human-checkpoint-batch). No PR opened, no acceptance-register edit (per scope — register edit is step 8/#2985).

Flipped successor #2993 to Agent Todo before this receipt. Stopped this worktree's dev server, API server, and TTS sidecar processes after the run.
